frontend/src/pages/articles/components/article-card/styles.module.scss

```diff
@@ -72,14 +73,29 @@
   line-height: normal;
 }
 
+.text ul {
+  padding-left: 40px;
+}
+
 .body .text {
-  margin: 15px 0 auto;
+  --max-lines: 3;
+  --height: 51px;
+
+  display: -webkit-box;
+  -webkit-line-clamp: var(--max-lines);
+  -webkit-box-orient: vertical;
+  max-width: 585px;
+  height: var(--height);
+  margin: 14px 0 23px;
+  overflow: hidden;
```

Comment:
Let's move text-overflow styling to `scaffolding` styles file as it's done for `visually-hidden` styling (we may also use text overflowing on overlap in other places, so it could be common)

Reply:
I think it will be good to create text-overflow styling for multiline and single-line (using for prompt tags, for example) in this case. Should I add `line-height: calc(var(--height) / var(--max-lines));` to text-overflow styling, so the texts aren't being cut off in case of fixed height?

Reply:
`single/multi` line cases are more specific, so maybe let's leave them in specific styling

Reply:
So, in scaffolding styles, I should only add this class?
```
.text-overflow {
    overflow: hidden;
    text-overflow: ellipsis;
}
```

Reply:
Yep